tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/CoreUtil.java

```diff
@@ -249,6 +249,20 @@ public static IFolder getDefaultDocrootFolder( IProject project )
         {
             retval = webproject.getDefaultDocrootFolder();
         }
+        else
+        {
+            final IBundleProject bundleProject = LiferayCore.create( IBundleProject.class, project );
```

Comment:
We should make the war-based gradle plugins respond to the `IWebProject` interface when it is asked for instead of looking for IBundleProject and then grabbing 'src/main/webapp'.